lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -65,6 +66,21 @@ def create_vm(attrs)
             vms_service.add(new_vm)
           end
           # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+          # rubocop:disable Metrics/AbcSize
+          def update_os_attrs(attrs)
+            attrs[:os] ||= {}
+            attrs[:os][:type] ||= attrs[:os_type] || "unassigned"
+            attrs[:os][:boot] ||= [attrs.fetch(:boot_dev1, OvirtSDK4::BootDevice::NETWORK), attrs.fetch(:boot_dev2, OvirtSDK4::BootDevice::HD)]
+            if attrs[:first_boot_dev]
+              attrs[:os][:boot] = attrs[:os][:boot].sort_by.with_index do |device, index|
```

Comment:
Two things:
1. We can use [`#find_index`](https://ruby-doc.org/core-2.2.0/Array.html#method-i-find_index) instead of `sort_by.with_index`
2. Do we expect the user to pass `attrs[:os][:boot]` list? I think it would be empty otherwise.

Reply:
1. I'll change it
2. we fill it on live 74

Reply:
find_index does not do what I want, this reorders the list so `first_boot_dev` is first